src/main/java/gov/cabinetofice/gapuserservice/web/UserController.java

```diff
@@ -46,6 +46,11 @@ public ResponseEntity<UserDto> getUserById(HttpServletRequest httpRequest, @Path
     @PatchMapping("/user/{userId}/department")
     public ResponseEntity<User> updateDepartment(HttpServletRequest httpRequest, @PathVariable("userId") Integer userId,
                                                    @RequestParam(value = "departmentId", required = false) Integer departmentId) {
+
+        if(oneLoginUserService.isUserApplicantAndFindOnly(oneLoginUserService.getUserById(userId))) {
+            throw new ForbiddenException();
```

Comment:
I think this should just be a 400 instead of a 403, since the client is permitted to perform this action, it's just not allowed for this user

Reply:
Fixed in latest commit